Upstream: yes

From 385df90a29e4f95aa6b128c9ce0df7344af14923 Mon Sep 17 00:00:00 2001
From: Michael Palimaka <kensington@gentoo.org>
Date: Wed, 2 Apr 2014 04:47:11 +1100
Subject: [PATCH] Fix build without X11.

If X11 was not found, HAVE_X11 would be unset. When substituted,
it resulted in literal "#if " causing build failure because of no
condition.

REVIEW: 117309
---
 CMakeLists.txt           | 1 +
 config-ksysguard.h.cmake | 2 +-
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 35acb25..5aa86ab 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -49,6 +49,7 @@ if(X11_FOUND)
     endif()
 endif()
 
+set(HAVE_X11 ${X11_FOUND})
 set(HAVE_XRES ${X11_XRes_FOUND})
 configure_file(config-ksysguard.h.cmake ${CMAKE_CURRENT_BINARY_DIR}/config-ksysguard.h )
 
diff --git a/config-ksysguard.h.cmake b/config-ksysguard.h.cmake
index 9928bc1..d95e1f5 100644
--- a/config-ksysguard.h.cmake
+++ b/config-ksysguard.h.cmake
@@ -11,4 +11,4 @@
 #cmakedefine HAVE_XRES 1
 
 /* Define if you have X11 at all */
-#define HAVE_X11 ${X11_FOUND}
+#cmakedefine01 HAVE_X11
-- 
1.9.1

